CHANGELOG.md

```diff
@@ -94,6 +94,7 @@
   have a wrong indentation.
 - Fixed a bug where on windows local packages had absolute paths in the manifest
   instead of relative.
+- avoid panicking in find_project_paths() when get_current_directory() returns Err.
```

Comment:
The changelog is for Gleam compiler users benefit not Gleam compiler contributors, so this needs to explain the change in terms of user-facing behaviour rather than compiler implementation details. And a capital letter at the start of the sentence please 😁